Duplicati/Library/Main/Operation/ListFilesHandler.cs

```diff
@@ -177,17 +193,17 @@ orderby n.Key
         public static List<KeyValuePair<long, Volumes.IParsedVolume>> ParseAndFilterFilesets(IEnumerable<Duplicati.Library.Interface.IFileEntry> rawlist, Options options)
         {
             var parsedlist = (from n in rawlist
-                        let p = Volumes.VolumeBase.ParseFilename(n)
-                        where p != null && p.FileType == RemoteVolumeType.Files
-                        orderby p.Time descending
-                        select p).ToArray();
+                              let p = Volumes.VolumeBase.ParseFilename(n)
+                              where p != null && p.FileType == RemoteVolumeType.Files
+                              orderby p.Time descending
+                              select p).ToArray();
             var filelistFilter = RestoreHandler.FilterNumberedFilelist(options.Time, options.Version);
-            return filelistFilter(parsedlist).ToList();                
+            return filelistFilter(parsedlist).ToList();
         }
-        
+
         public static IEnumerable<Library.Interface.IListResultFileset> CreateResultSequence(IEnumerable<KeyValuePair<long, Volumes.IParsedVolume>> filteredList)
         {
-            return (from n in filteredList select (Library.Interface.IListResultFileset)(new ListResultFileset(n.Key, n.Value.Time.ToLocalTime(), -1, -1))).ToArray();
-        }        
+            return (from n in filteredList select (Library.Interface.IListResultFileset)(new ListResultFileset(n.Key, 0, n.Value.Time.ToLocalTime(), -1, -1))).ToArray();

```

Comment:
Should this also be `BackupType.PARTIAL_BACKUP` instead of `0`?

Reply:
Also, why are these considered "partial"?  Or, is it that the backup type doesn't matter and we have to choose _some_ value?

Sorry, I'm not that familiar with this part of the code...

Reply:
> BackupType.PARTIAL_BACKUP

Yes.. it should be.

Reply:
> Also, why are these considered "partial"? Or, is it that the backup type doesn't matter and we have to choose _some_ value?
> 
> Sorry, I'm not that familiar with this part of the code...

It's the beginning for that object and the fileset so we default it to being a partial backup. It only becomes a full backup when the backup is completed without interruption.

Reply:
Got it.  Thanks!